trace/spanstore.go

```diff
@@ -29,8 +31,15 @@ var (
 	spanStores = make(map[string]*spanStore)
 )
 
+// This exists purely to avoid exposing internal methods used by z-Pages externally.
+type internalOnly struct{}
```

Comment:
type reporter struct{}

Reply:
in light of the previous comment about being unable to really create a strongly typed interface, do you still think this should be renamed? I kind of prefer it as it is because it's definitely just a placeholder, not trying to pretend that it's a real abstraction (that's future work) - it really is just trying to hide the functions in the simplest way possible.

Reply:
I am fine with the things the way they are.